crates/cubecl-linalg/src/matmul/components/global/args.rs

```diff
@@ -60,6 +61,91 @@ pub struct TensorInput<EG: Numeric, GA: MatmulArgs<EG>> {
     ident: TensorInputIdent,
 }
 
+impl<EG: Numeric, MA: MatmulArgs<EG>> VirtualTensorOperations<EG> for TensorInput<EG, MA> {}
+impl<EG: Numeric, MA: MatmulArgs<EG>> VirtualTensorOperations<EG> for TensorOutput<EG, MA> {}
+
+impl<EG: Numeric, MA: MatmulArgs<EG>> VirtualTensorOperationsExpand<EG>
+    for TensorOutputExpand<EG, MA>
```

Comment:
The ordering of `impl` blocks in this file is really inconsistent. I suggest either implementing everything related `TensorInput` than everything related to `TensorOuput` or to implement similar functionalities for both consecutively for each functionality. But not intertwined like it currently is.